client/blocks/product-selector/index.jsx

```diff
@@ -499,6 +508,22 @@ export class ProductSelector extends Component {
 		return null;
 	}
 
+	getPromo() {
```

Comment:
How about the following method name?

```suggestion
	renderPromo() {
```